app/controllers/events_controller.rb

```diff
@@ -0,0 +1,29 @@
+class EventsController < ApplicationController
+  before_action :authenticate_user!, except: %i[index]
+
+  def index
+    @events = Event.all
+    @future_events = @events.upcoming
+    @past_events = @events.past
+  end
+
+  def create
+    @event = current_user.events.create(user_params)
```

Comment:
- [ ] uses #build method on association, i.e. `current_user.events.build(event_params)` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)).
Kindly edit the highlighted line of code to reflect :point_up: